crossbar/adapter/rest/test/request_mock.py

```diff
@@ -47,16 +47,16 @@ def _render(resource, request):
         raise ValueError("Unexpected return value: %r" % (result,))
 
 
-def _requestMock(path, method="GET", host="localhost", port=8080, isSecure=False,
+def _requestMock(path, method=b"GET", host=b"localhost", port=8080, isSecure=False,
                  body=None, headers=None, args=None, reactor=None):
 
     if not headers:
         headers = {}
 
-    headers["Date"] = ["Tue, 01 Jan 2014 01:01:01 GMT"]
+    headers[b"Date"] = ["Tue, 01 Jan 2014 01:01:01 GMT"]
```

Comment:
Good catch... nothing in Crossbar uses the HTTP date so this slipped through, I'll fix it next py3 pass.